spec/features/trainer_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+require 'support/helpers/trainer_helper.rb'
+include TrainerHelper
+
+describe "training without cards" do
+  before :each do
+    create(:card)
+  end
+
+  it 'no cards' do
+    visit root_path
+    expect(page).to have_content 'Ожидайте наступления даты пересмотра.'
+  end
+end
+
+describe "training with some cards" do
+  before :each do
+    create_review_cards
+  end
+
+  it 'first visit' do
+    visit root_path
+    expect(page).to have_content 'Original text'
+  end
+
+  it 'incorrect translation' do
+    check_translation('RoR')
+    expect(page).
+        to have_content 'Вы ввели не верный перевод. Повторите попытку.'
```

Comment:
Use 2 (not 4) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.